src/pages/RegisterPage.js

```diff
@@ -0,0 +1,201 @@
+import React, { useRef } from "react";
+import { useForm } from "react-hook-form";
+import "../assets/styles/RegisterForm.css";
+import { PrimaryButton } from "../components/PrimaryButton";
+import axios from "axios";
+
+function RegisterPage() {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    watch,
+  } = useForm({ mode: "onBlur" });
+
+  const password = useRef({});
+  password.current = watch("password", "");
+
+  const onSubmit = (data, e) => {
+    e.target.reset();
+    console.log(data);
```

Comment:
Please remove this console.log

Reply:
Thank you Leonel! I removed unnecessary console log.